src/zope/browserpage/metaconfigure.py

```diff
@@ -168,6 +168,7 @@ def page(_context, name, permission, for_=Interface,
                                required)
 
     _handle_for(_context, for_)
+    new_class._simple_whitelist = set(required) - set([attribute, 'browserDefault', '__call__', 'publishTraverse'])
```

Comment:
Maybe this should be `_simple__whitelist` with the double underscore that you'd get if you used `self.__simple` inside the `simple` class?  To prevent clashes with arbitrary subclasses etc.

Reply:
Probably a good idea.

Reply:
done in https://github.com/zopefoundation/zope.browserpage/commit/dcdda5f3fa0ad4f9f79ce0cf593c33f548f083b5 and https://github.com/zopefoundation/Zope/pull/643/commits/6c1134448fed1bfc25076f3738c977b541ce8ce4